go-controller/pkg/types/const.go

```diff
@@ -100,6 +100,9 @@ const (
 	GatewayLBUDP          = "UDP_lb_gateway_router"
 	GatewayLBSCTP         = "SCTP_lb_gateway_router"
 
+	// Loadbalancer options
```

Comment:
maybe a better comment including some of the words from your description about why this is useful.

Reply:
ack